tests/foreman/cli/test_registration.py

```diff
@@ -334,3 +334,49 @@ def test_positive_global_registration_with_gpg_repo(
     assert result.status == 0
     if not is_open('SAT-27653'):
         assert rhel_contenthost.execute('dnf install -y bear').status == 0
+
+
+@pytest.mark.parametrize('setting_update', ['default_location_subscribed_hosts'], indirect=True)
+@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+def test_positive_verify_default_location_for_registered_host(
+    module_target_sat,
+    module_sca_manifest_org,
+    rhel_contenthost,
+    module_activation_key,
+    setting_update,
+):
+    """Test default location set for registered host.
+
+    :id: 6ea802d8-0788-4309-845c-c877013a8e48
+
+    :steps:
+        1. Create a location and set it as a default location in Administer --> Settings --> Content --> "Default Location subscribed hosts".
+        2. Register the host without specifying the location.
+
+    :expectedresults: Verify that the default location in settings is set as host location after registration.
+
+    :Verifies: SAT-23047
+
+    :customerscenario: true
+    """
+    org = module_sca_manifest_org
+    location = module_target_sat.api.Location(organization=[org]).create()
+    setting_update.value = location.name
+    setting_update.update({'value'})
```

Comment:
I think default when org is created it would have Default location, so why not use a new location here instead?

Reply:
I have used a new location here. I have created a location in the organization. Or did I misunderstood the question?

Reply:
No. The default location is set to the location I created in the test.